features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/stored-procedures/oracle/sessiondata-cleanup/oracle-sessiondata-cleanup.sql

```diff
@@ -127,14 +137,49 @@ BEGIN
             END IF;
 
             -- Deleting user-session mappings from 'IDN_AUTH_USER_SESSION_MAPPING' table
-            EXECUTE IMMEDIATE 'DELETE FROM IDN_AUTH_USER_SESSION_MAPPING WHERE SESSION_ID IN (SELECT SESSION_ID FROM TEMP_SESSION_BATCH)';
-            sessionmappingcleanupcount := SQL%rowcount;
-            COMMIT;
+            SELECT COUNT(*) INTO ROWCOUNT from ALL_TABLES where OWNER = CURRENT_SCHEMA AND table_name = upper('IDN_AUTH_USER_SESSION_MAPPING');
+            IF (ROWCOUNT = 1) then
+                EXECUTE IMMEDIATE 'DELETE FROM IDN_AUTH_USER_SESSION_MAPPING WHERE SESSION_ID IN (SELECT SESSION_ID FROM TEMP_SESSION_BATCH)';
+                sessionmappingcleanupcount := SQL%rowcount;
+                COMMIT;
 
-            IF ( tracingenabled ) THEN
-            EXECUTE IMMEDIATE 'INSERT INTO LOG_WSO2_SESSION_CLEANUP_SP (TIMESTAMP,LOG) VALUES (TO_CHAR( SYSTIMESTAMP, ''DD.MM.YYYY HH24:MI:SS:FF4''),''DELETED USER-SESSION MAPPINGS COMPLETED WITH '|| sessionmappingcleanupcount||''')';
-            COMMIT;
-            END IF;
+                IF ( tracingenabled ) THEN
```

Comment:
remove the spaces

Reply:
Check other places as well

Reply:
Addressed in https://github.com/wso2/carbon-identity-framework/pull/3178/commits/3d2233c7745ad78dc3f6e7e384b163ed21b9ddaa